src/plugins/data/common/search/aggs/agg_config.ts

```diff
@@ -172,6 +174,31 @@ export class AggConfig {
     return _.get(this.params, key);
   }
 
+  hasTimeShift(): boolean {
+    return Boolean(this.getParam('timeShift'));
+  }
+
+  getTimeShift(): undefined | moment.Duration {
```

Comment:
Something in this logic seems to be taking the absolute value of the time shift, so negative values are ignored. For example `-60m` is the same as `1h` based on what I was seeing. I like this behavior- makes it impossible to make a mistake- but it was a little surprising. Can you add a test for this?

Reply:
This happened on accident - what do you think about treating this as an invalid interval?